stratego.build/src/main/java/mb/stratego/build/strincr/SubFrontend.java

```diff
@@ -133,12 +130,12 @@ public Output(IStrategoTerm result) {
         }
     }
 
-    private final IResourceService resourceService;
+    private final IOAgentTrackerFactory ioAgentTrackerFactory;
     private final StrIncrContext strContext;
-    ILanguageImpl strategoLang;
+    //ILanguageImpl strategoLang; // GK: removed, does not seem to be used.
```

Comment:
Probably a holdover from a refactoring. The only field like this that was actually used was in the `Frontend` class. Just remove the line.